From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: gellert <gellert.hegyi@around.co>
Date: Thu, 18 Jun 2020 15:50:11 +0200
Subject: removes screen capture cpu limitation

This patch disables the frame late limitation exposed to screen capture.

Signed-off-by: gellert <gellert.hegyi@around.co>

diff --git a/content/browser/media/capture/desktop_capture_device.cc b/content/browser/media/capture/desktop_capture_device.cc
index a30e05a9aac9c6d14430be92ee2824d423e7e26d..cdbac5d5e7c9734fa4a18f82e0c37cbc52c813de 100644
--- a/content/browser/media/capture/desktop_capture_device.cc
+++ b/content/browser/media/capture/desktop_capture_device.cc
@@ -481,9 +481,7 @@ void DesktopCaptureDevice::Core::ScheduleNextCaptureFrame() {
   base::TimeDelta last_capture_duration = NowTicks() - capture_start_time_;
 
   // Limit frame-rate to reduce CPU consumption.
-  base::TimeDelta capture_period =
-      std::max((last_capture_duration * 100) / max_cpu_consumption_percentage_,
-               requested_frame_duration_);
+  base::TimeDelta capture_period = requested_frame_duration_;
 
   // Schedule a task for the next frame.
   capture_timer_->Start(FROM_HERE, capture_period - last_capture_duration, this,
